package.json

```diff
@@ -107,8 +107,7 @@
   "lint-staged": {
     "{src,spec}/**/*.js": [
       "prettier --write",
-      "flow",
```

Comment:
if you're removing this, should @Flow's come out too?

Reply:
Yeah, flow doesn’t take an argument, i’ll Find a way to add it back